From 62bbd3b664d4b03011a4b382ed20353a91c30406 Mon Sep 17 00:00:00 2001
From: Ariadne Conill <ariadne@dereferenced.org>
Date: Tue, 21 Jan 2020 10:32:36 -0600
Subject: [PATCH] cli: remove --version to --modversion remapping

This has been a source of frequent complaints, so we drop it.
Resolves: https://todo.sr.ht/~kaniini/pkgconf/6
---
 cli/main.c | 14 ++------------
 1 file changed, 2 insertions(+), 12 deletions(-)

diff --git a/cli/main.c b/cli/main.c
index 563ec8f0cfcd..fc698a4f9191 100644
--- a/cli/main.c
+++ b/cli/main.c
@@ -1005,18 +1005,8 @@ main(int argc, char *argv[])
 
 	if ((want_flags & PKG_VERSION) == PKG_VERSION)
 	{
-		if (argc > 2)
-		{
-			fprintf(stderr, "%s: --version specified with other options or module names, assuming --modversion.\n", argv[0]);
-
-			want_flags &= ~PKG_VERSION;
-			want_flags |= PKG_MODVERSION;
-		}
-		else
-		{
-			version();
-			return EXIT_SUCCESS;
-		}
+		version();
+		return EXIT_SUCCESS;
 	}
 
 	if ((want_flags & PKG_HELP) == PKG_HELP)
